src/v2/Apps/Partner/Components/Overview/ArticlesRail.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react"
+import { Box, Flex, Sans, Text } from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
+import { Carousel } from "v2/Components/Carousel"
+import { ArticleCardFragmentContainer as ArticleCard } from "v2/Apps/Partner/Components/PartnerArticles/ArticleCard"
+import { data as sd } from "sharify"
+import { ArticlesRail_articles } from "v2/__generated__/ArticlesRail_articles.graphql"
+
+interface ArticlesRailProps {
+  articles: ArticlesRail_articles
+  partnerSlug: string
+}
+
+const ArticlesRail: React.FC<ArticlesRailProps> = ({
+  articles,
+  partnerSlug,
+}) => {
+  return (
+    <>
+      <Flex justifyContent="space-between" alignItems="center">
+        <Text variant="title">Articles</Text>
+        <Sans fontSize={14} size="2" color="black60">
```

Comment:
`Sans` is deprecated — only use `Text` and [the v2 variants](https://palette-storybook.artsy.net/?path=/story/components-text--variants)

Reply:
fixed